app/queries/decidim/action_delegator/published_responses.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ActionDelegator
+    class PublishedResponses < Rectify::Query
+      def initialize(consultation)
+        @consultation = consultation
+      end
+
+      # The Question's default_scope, `order(order: :asc)`, messes up the ordering in our queries so
+      # we have to explicitly remove the ORDER BY close using `#reorder`.
```

Comment:
:fire: I gave up on reusing the `Question.published` scope which, oddly, included the ORDER statement anyway. Merging this with `ResponsesByMembership` was simply removing all ordering :see_no_evil: